### PR TITLE
Add translations for footer and answer items

### PR DESCRIPTION
**Pull request by nastikue**
Add translations in Footer, AnswerItem components

## Motivation

Not all text on PatientBoard was translated

## Implementation

1. patientBoard.json for all languages was changed: added translation for Footer
2. common.json for all languages added with translation for AnswerItem 
3. redundant about.json and physicianView.json removed

## Screenshots/Links

image

### CODE REVIEW

File: client/public/locales/de/common.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "showMore": "[DE] Show more",
+  "showLess": "[DE] Show less",
+  "answeredBy": "[DE] Answered By <0>{{firstAnsweredBy}}</0> on <1>{{firstAnsweredOn}}</1>",
```
Comment (aviskarkc10) [inline client/public/locales/de/common.json:4]: @nastikue what do the tags here mean? `<0></0>`?

Comment (aviskarkc10) [reply client/public/locales/de/common.json:4]: are they placeholders for html tags? If they are, I got it.

File: client/src/components/Footer.js [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 
 import '../styles/Footer.css';
+import { withTranslation, Trans } from 'react-i18next';
 
 const Footer = () => {
   const disclaimerLink =
     'https://nquestionblob.blob.core.windows.net/images/' +
     'Full%20Disclaimer%20_%20Legal%20Information%20and%20Disclosures_%20Nth%20Opinion.pdf';
 
+  /* eslint-disable jsx-a11y/anchor-has-content */
```
Comment (aviskarkc10) [inline client/src/components/Footer.js:11]: why do we need this?

File: client/public/locales/de/common.json (hunk 1 above)
Comment (nastikue) [reply client/public/locales/de/common.json:4]: @aviskarkc10 , yes these are placeholders for html tags that are used for i18n Trans component text interpolation https://react.i18next.com/latest/trans-component

File: client/src/components/Footer.js (hunk 2 above)
Comment (nastikue) [reply client/src/components/Footer.js:11]: Without this disable changes with empty anchor couldn't be committed.

Comment (sethuraj) [reply client/src/components/Footer.js:11]: Hi, resolved the conversation? If so I can merge. Thanks

Comment (nastikue) [reply client/src/components/Footer.js:11]: @aviskarkc10 , can this PR be merged?

Comment (aviskarkc10) [reply client/src/components/Footer.js:11]: @nastikue merged. Sorry hadn't paid attention to this.

Comment (nastikue) [reply client/src/components/Footer.js:11]: @aviskarkc10 great, thank you!

**Pull request merged**